test/redfish-core/include/redfish_aggregator_test.cpp

```diff
@@ -1,3 +1,4 @@
+#ifdef BMCWEB_ENABLE_REDFISH_AGGREGATION
```

Comment:
Why do this?

Reply:
otherwise this test code compiles and fails.
this test code needs to run with BMCWEB_ENABLE_REDFISH_AGGREGATION enabled